test/test.js

```diff
@@ -238,24 +238,25 @@ describe('ipfs node api', function () {
     // naja
     assert(id.ID)
     assert(id.PublicKey)
-  }),
+  })
 
-  it("returns an error when getting a non-existent key from the DHT",
+  it('returns an error when getting a non-existent key from the DHT',
     function (done) {
-    ipfs.dht.get('non-existent', function (error, value) {
-      assert(error);
-      done();
-    });
-  });
+      ipfs.dht.get('non-existent', function (error, value) {
+        assert(error)
+        done()
+      })
+    })
 
-  it("puts and gets a key value pair in the DHT", function (done) {
-    this.timeout(5 * 1000);
+  it('puts and gets a key value pair in the DHT', function (done) {
+    this.timeout(5 * 1000)
```

Comment:
oh maybe it is that the timeout in the daemon is not hitting before the timeout in the test.

we can pass the new --timeout option to the call:

```
ipfs dht put --timeout 100ms scope interplanetary
```